EventFilter/CTPPSRawToDigi/python/ctppsDigiToRaw_cff.py

```diff
@@ -0,0 +1,17 @@
+import FWCore.ParameterSet.Config as cms
+from EventFilter.RawDataCollector.rawDataCollector_cfi import *
+
+from EventFilter.CTPPSRawToDigi.ctppsPixelRawData_cfi import ctppsPixelRawData
+from EventFilter.CTPPSRawToDigi.ctppsTotemRawData_cfi import ctppsTotemRawData
+
+ctppsRawData = cms.Sequence()
```

Comment:
Please use `cms.Task()` instead of `cms.Sequence()` in this file (that should fix the matrix/addOn errors).

Reply:
Done